### DISCUSSION TITLE
allow additional attributes with 'Extra.allow'

### DISCUSSION THREAD

**Discussion by sysid** [koxudaxi/datamodel-code-generator#734] [Q&A] [answered]
I maybe overlooking something, but I could not find a way to accomplish it:

Can I somehow allow extra attributes on the models?

```
    class Config:
        extra = Extra.allow
```

Currently I just to a search/replace after schema creation, but I figure this requirement is probably not uncommon and I am just missing something?

In any way, this tool is very helpful! Thanks for creating it.

**Comment by koxudaxi** [comment, marked as answer]
@sysid 
I'm sorry for my too-late reply.
We should add `"additionalProperties": true`
we can get `Extra.allow` in the model.
https://json-schema.org/understanding-json-schema/reference/object.html#additional-properties
